tests/installation/partitioning_lvm.pm

```diff
@@ -100,15 +100,15 @@ sub run {
             assert_screen "inst-select-root-disk";
             send_key 'alt-n';
         }
-        if (leap_version_at_least('15.0')) {
+        if (leap_version_at_least('15.0') || is_tumbleweed) {
```

Comment:
Would the following work?: `leap_version_at_least('15.0') || !is_sle`?

Reply:
Yeah, I changed it to that, due to `!leap_version_at_least('15.0')` is valid if TW, this probably is the only option if preference TW as default.